test/controllers/api/auth/feeds_controller_test.rb

```diff
@@ -102,18 +102,18 @@ class << @controller; attr_accessor :prx_auth_token; end
         _(feed.feed_images.count).must_equal 0
         fua = feed.updated_at
 
-        update_hash = { feedImage: { href: url1, description: 'd1' } }
+        update_hash = { feedImage: { href: url1, caption: 'd1' } }
```

Comment:
Some tests were using now gone fields to track which image is which in the API.

Switched them to fields that exist after these migrations.